source/includes/admonitions/connection-resets.rst

```diff
@@ -0,0 +1,9 @@
+.. important::
+
+   Methods which reset the shell's connection to the server trigger a
+   termination of all open :manual:`sessions </reference/server-sessions>`.
```

Comment:
I think we should specify whether we mean client or server sessions.

Reply:
Done.